src/main/java/frc/robot/subsystems/intake/IntakeLoggedInputs.java

```diff
@@ -0,0 +1,27 @@
+package frc.robot.subsystems.intake;
+
+import org.littletonrobotics.junction.LogTable;
+import org.littletonrobotics.junction.inputs.LoggableInputs;
+
+public class IntakeLoggedInputs implements LoggableInputs {
+    public double motorPower;
+    public boolean leftSolenoidState;
+    public boolean rightSolenoidState;
```

Comment:
What does this mean?!?!?! What does the state mean?!?!?!?!

Reply:
[The Fitness Gram™ Pacer Test](https://www.youtube.com/watch?v=Y82jDHRrswc)
The FitnessGram™ Pacer Test is a multistage aerobic capacity test that progressively gets more difficult as it continues. The 20 meter pacer test will begin in 30 seconds. Line up at the start. The running speed starts slowly, but gets faster each minute after you hear this signal. [beep] A single lap should be completed each time you hear this sound. [ding] Remember to run in a straight line, and run as long as possible. The second time you fail to complete a lap before the sound, your test is over. The test will begin on the word start. On your mark, get ready, start.


Also, happy hannukah

Reply:
no more Solenoids!!!!